third_party/aog-payjs/payjs.js

```diff
@@ -0,0 +1,2077 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import uuid from './third_party/random_uuid/Random.uuid.js';
```

Comment:
These should be bundled too -- I believe SwG uses rollup to do this. 

@dvoytenko Is there a rollup config you can share?

Reply:
The rollup config SwG uses is: https://github.com/subscriptions-project/swg-js/blob/master/build-system/tasks/export-to-es.js#L88